.github/workflows/e2e-compatibility-workflow-call.yaml

```diff
@@ -2,15 +2,14 @@ on:
   workflow_call:
     inputs:
       docker-tag:
-        description: 'Docker tag being used'
+        description: "Docker tag being used"
```

Comment:
I'd prefer not to include these changes unless they provide some benefit?

Reply:
Sorry that is my linter on save.